EIPS/eip-2541.md

```diff
@@ -0,0 +1,273 @@
+---
+eip: 2541
+title: EY SW6-Bis curve operations
+author: Alex Vlasov (@shamatar)
+discussions-to: https://github.com/ethereum/EIPs/pull/2541
```

Comment:
This cannot be the Pull Request.  You can create a thread on https://ethereum-magicians.org/ or you can create an issue here on GitHub.